ui/hooks/useDisplayName.ts

```diff
@@ -9,22 +9,24 @@ import { useName } from './useName';
  * @param value
  * @param type
  * @param variation
- * @returns The name if it can be resolved, otherwise null.
+ * @returns An object with two properties:
+ * - `name` {string|null} - The display name, if it can be resolved, otherwise null.
+ * - `hasPetname` {boolean} - True iff there is a petname for the given address.
```

Comment:
```suggestion
 * - `hasPetname` {boolean} - True if there is a petname for the given address.
```

Reply:
"iff" is short for "if and only if". https://en.wikipedia.org/wiki/If_and_only_if. 

That might be too jargony for comments like this though. Done.